2021-03-24 09:22:01,046 INFO    MainThread:14188 [wandb_setup.py:_flush():69] setting env: {}
2021-03-24 09:22:01,046 INFO    MainThread:14188 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-24 09:22:01,046 INFO    MainThread:14188 [wandb_init.py:_log_setup():334] Logging user logs to /Users/shervinazadi/Documents/Git/MASS_DQN/notebooks/tut_03_dqn_atari/wandb/run-20210324_092201-lbitqim4/logs/debug.log
2021-03-24 09:22:01,046 INFO    MainThread:14188 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/shervinazadi/Documents/Git/MASS_DQN/notebooks/tut_03_dqn_atari/wandb/run-20210324_092201-lbitqim4/logs/debug-internal.log
2021-03-24 09:22:01,047 INFO    MainThread:14188 [wandb_init.py:_jupyter_setup():285] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f8ee8947370>
2021-03-24 09:22:01,047 INFO    MainThread:14188 [wandb_init.py:init():367] calling init triggers
2021-03-24 09:22:01,047 INFO    MainThread:14188 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-03-24 09:22:01,048 INFO    MainThread:14188 [wandb_init.py:init():416] starting backend
2021-03-24 09:22:01,048 INFO    MainThread:14188 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-03-24 09:22:01,064 INFO    MainThread:14188 [backend.py:ensure_launched():123] starting backend process...
2021-03-24 09:22:01,072 INFO    MainThread:14188 [backend.py:ensure_launched():127] started backend process with pid: 22965
2021-03-24 09:22:01,076 INFO    MainThread:14188 [wandb_init.py:init():421] backend started and connected
2021-03-24 09:22:01,154 INFO    MainThread:14188 [wandb_init.py:init():461] updated telemetry
2021-03-24 09:22:01,154 INFO    MainThread:14188 [wandb_init.py:init():480] communicating current version
2021-03-24 09:22:01,791 INFO    MainThread:14188 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.23 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-03-24 09:22:01,792 INFO    MainThread:14188 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-24 09:22:01,969 INFO    MainThread:14188 [wandb_init.py:init():518] starting run threads in backend
2021-03-24 09:22:01,985 INFO    MainThread:14188 [wandb_run.py:_console_start():1480] atexit reg
2021-03-24 09:22:01,986 INFO    MainThread:14188 [wandb_run.py:_redirect():1343] redirect: SettingsConsole.WRAP
2021-03-24 09:22:01,986 INFO    MainThread:14188 [wandb_run.py:_redirect():1378] Wrapping output streams.
2021-03-24 09:22:01,986 INFO    MainThread:14188 [wandb_run.py:_redirect():1394] Redirects installed.
2021-03-24 09:22:01,986 INFO    MainThread:14188 [wandb_init.py:init():542] run started, returning control to user process
2021-03-24 09:22:02,033 INFO    MainThread:14188 [wandb_init.py:_pause_backend():247] pausing backend
2021-03-24 09:22:02,033 INFO    MainThread:14188 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-03-24 09:22:30,384 INFO    MainThread:14188 [wandb_init.py:_resume_backend():256] resuming backend
2021-03-24 09:22:30,400 INFO    MainThread:14188 [wandb_setup.py:_flush():69] setting env: {}
2021-03-24 09:22:30,400 INFO    MainThread:14188 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-24 09:22:30,400 INFO    MainThread:14188 [wandb_init.py:_log_setup():334] Logging user logs to /Users/shervinazadi/Documents/Git/MASS_DQN/notebooks/tut_03_dqn_atari/wandb/run-20210324_092230-2rd055vf/logs/debug.log
2021-03-24 09:22:30,401 INFO    MainThread:14188 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/shervinazadi/Documents/Git/MASS_DQN/notebooks/tut_03_dqn_atari/wandb/run-20210324_092230-2rd055vf/logs/debug-internal.log
2021-03-24 09:22:30,401 INFO    MainThread:14188 [wandb_init.py:init():367] calling init triggers
2021-03-24 09:22:30,401 INFO    MainThread:14188 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-03-24 09:22:30,401 INFO    MainThread:14188 [wandb_init.py:init():387] re-initializing run, found existing run on stack: lbitqim4
2021-03-24 09:22:30,402 INFO    MainThread:14188 [wandb_run.py:finish():1139] finishing run shervinazadi/dqn-tut-03-test/lbitqim4
